### PR TITLE
Fix/ Submission Revision: add submission readers as edit readers

### PR DESCRIPTION
**Pull request by celestemartinez**

### CODE REVIEW

File: tests/test_venue_request_v2.py [hunk 1]
```diff
@@ -3264,27 +3270,20 @@ def test_supplementary_material_revision(self, client, test_client, selenium, re
...
+        assert revision_invitation.edit['readers'] == ['V2.cc/2030/Conference',
+            'V2.cc/2030/Conference/Submission3/Senior_Area_Chairs',
+            'V2.cc/2030/Conference/Submission3/Area_Chairs',
+            'V2.cc/2030/Conference/Submission3/Reviewers',
+            'V2.cc/2030/Conference/Submission3/Authors']
 
         assert all(x not in revision_invitation.edit['note']['content'] for x in ['title','authors', 'authorids','abstract','keywords', 'TLDR'])
-        assert 'supplementary_material' in revision_invitation.edit['note']['content']
-
-#         #make sure homepage webfield was not overwritten after doing get_conference()
```
Comment (melisabok) [inline tests/test_venue_request_v2.py:3275]: are the tests failing here?

Comment (celestemartinez) [reply tests/test_venue_request_v2.py:3275]: I think we copied this over from `test_venue_request.py` when we starting creating API2 tests. I don't think it applies to this test but I can try to adapt it.

Comment (melisabok) [reply tests/test_venue_request_v2.py:3275]: let's remove the commented code then

Comment (celestemartinez) [reply tests/test_venue_request_v2.py:3275]: yep I removed it in this PR already

**Review by melisabok — APPROVED** [review]

**Pull request merged**